src/components/Landing/EducationalSection/EducationalImage.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, Image } from "@mantine/core";
+import classes from "./EducationalSection.module.css";
+
+export function EducationImage({
+  imageSrc,
+  hoverImageSrc,
+}: {
+  imageSrc: string;
+  hoverImageSrc: string;
+}) {
+  return (
+    <Box
+      w={{ base: "100%", md: "45%" }}
+      p="md"
+      style={{ position: "relative" }}
+    >
+      <div className={classes.blurBackground} />
```

Comment:
you can use Box and style with mantine
use css for purposes that cannot be done with the library at hand, you have installed